app/code/core/Mage/Payment/Block/Form/Container.php

```diff
@@ -45,8 +45,8 @@ protected function _prepareLayout()
          */
         foreach ($this->getMethods() as $method) {
             $this->setChild(
-               'payment.method.'.$method->getCode(),
-               $this->helper('payment')->getMethodFormBlock($method)
+                'payment.method.'.$method->getCode(),
```

Comment:
Ideally, there should a space before and after the concatenation operator i.e : `'payment.method.' . $method->getCode(),`

Reply:
```suggestion
                'payment.method.' . $method->getCode(),
```